homeassistant/components/jellyfin/config_flow.py

```diff
@@ -32,6 +34,15 @@
 )
 
 
+OPTIONAL_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional("audio_codec", default="None"): vol.In(
+            ["None", *SUPPORTED_AUDIO_CODECS]
```

Comment:
```suggestion
        vol.Optional("audio_codec"): vol.In(
            SUPPORTED_AUDIO_CODECS
```

`None` will be returned if the user does not select anything